packages/dd-trace/src/appsec/telemetry.js

```diff
@@ -26,10 +27,19 @@ function disable () {
   enabled = false
 }
 
+function newStore () {
+  return {
+    [DD_TELEMETRY_REQUEST_METRICS]: {
```

Comment:
I'm not sure I understand why you're using a Symbol here ?

Reply:
To ensure uniqueness. I know the property is used internally by the module and there is no need to use a Symbol...
I can change it if you consider it inappropriate.

Reply:
That's fine, as long as you thought about the question, up to you